site/quickstart.md

```diff
@@ -0,0 +1,262 @@
+
+```bash
+imagetool create --tag wls:12.2.1.3.0 --latestPSU --version 12.2.1.3.0 --user username@mycompany.com --passwordEnv MYPWD  [--httpProxyUrl http://company-proxy:80 --httpsProxyUrl http://company-proxy:80]
+```
+
+where ```--user --passwordEnv``` provides the user credential who is entitled to download patches from Oracle Support
+
+You will see the docker command output as the tool runs:
+
+```bash
+[2019-05-28 10:37:02] [com.oracle.weblogic.imagetool.cli.menu.CreateImage] [INFO   ] tmp directory used for build 
+context: /home/acmeuser/wlsimgbuilder_temp8791654163579491583 
+[2019-05-28 10:37:09] [com.oracle.weblogic.imagetool.cli.menu.CreateImage] [INFO   ] docker cmd = docker build 
+--force-rm --rm=true --no-cache --tag wls:12.2.1.3.0 --build-arg http_proxy=http://company-proxy.com:80 --build-arg 
+https_proxy=http://company-proxy.com:80 --build-arg WLS_PKG=fmw_12.2.1.3.0_wls_Disk1_1of1.zip --build-arg 
+JAVA_PKG=jdk-8u201-linux-x64.tar.gz --build-arg PATCHDIR=patches /home/acmeuser/wlsimgbuilder_temp8791654163579491583 
+Sending build context to Docker daemon   1.08GB
+
+Step 1/46 : ARG BASE_IMAGE=oraclelinux:7-slim
+...
+Removing intermediate container 57ccb9fff56b
+ ---> 18d366fc3da4
+Successfully built 18d366fc3da4
+Successfully tagged wls:12.2.1.3.0
+```
+
+Once the image is created, you can use any docker commands on the image, for example:
+
+```bash
+docker images
+
+REPOSITORY          TAG                 IMAGE ID            CREATED              SIZE
+wls                 12.2.1.3.0          18d366fc3da4        About a minute ago   1.41GB
+oraclelinux         7-slim              f7512ac13c1b        6 weeks ago          118MB
+
+```
+
+## Creating image with no internet access
+
+
+In this use case, since there is no internet access.  You will be responsible for downloading all the installers and 
+patches, plus setting up the cache.  You also have to provide a base operating system image that has the following packages 
+installed.
+
+
+```bash
+gzip 
+tar 
+unzip
+```
+
+For each WebLogic patches, you will need to download it from Oracle Support and set up the cache. For example, if you downloaded patch number 
+27342434 for WebLogic version 12.2.1.3.0:
+ 
+```bash
+imagetool cache addPatch --patchId 27342434 --version 12.2.1.3.0 --path /home/acmeuser/cache/p27342434_122130_Generic .zip -user username@mycompany.com --passwordEnv MYPWD
+```
+
+You need to provide the credential in the command line, it verifies the MD5 on the file system against the meta data on 
+Oracle Support for this patch number
+
+Then, you can run the command to create the image:
+
+```bash
+imagetool create --fromImage myosimg:latest --tag wls:12.2.1.3.0 --patches 27342434 --version 12.2.1.3.0 --useCache always
+```
+
+Sometimes, a WebLogic patch may require patching the OPatch binaries before applying them.  We recommend 
+downloading the latest OPatch's patch and setup the cache.  For example, the latest OPatch's patch is 28186730, 13.9
```

Comment:
latest OPatch's patch and setup the cache.  For example, the latest OPatch's patch is ->
latest OPatch patch and set up the cache.  For example, the latest OPatch patch is